lark/parsers/earley_forest.py

```diff
@@ -459,15 +459,20 @@ class PackedData():
     that comes from the left child and the right child.
     """
 
+    class _NoData():
+        pass
+
+    NO_DATA = _NoData()
+
     def __init__(self, node, data):
-        self.left = None
-        self.right = None
+        self.left = self.NO_DATA
+        self.right = self.NO_DATA
         if data:
-            if node.left:
+            if node.left is not None:
```

Comment:
Why is this `None` and not `NO_DATA` ?

Reply:
`node` is a `PackedNode`. I am checking if `node` has a left child. `NO_DATA` has only been used for the `PackedData` class, not the `PackedNode` class.